### ISSUE TITLE
Add a (redesigned) Client UI

### ISSUE THREAD

**Issue by jdigger** [TWCable/grabbit#17]
There should be an easy way to start jobs from a pretty UI, as well as monitoring and management.

**Comment by paulrohrbeck** [comment]
I agree @jdigger, it would be great to have something in the Tools section of CQ/AEM. You probably don't have a road map for this feature, right?

**Comment by jdigger** [comment]
No roadmap. Focussed more on pure "functionality" right now, but if you have an itch... ;-)

**Comment by paulrohrbeck** [comment]
Cool. Yeah we are working on a basic UI. I just wanted to make sure we are not working on it at the same time. I'll let you know when we have something to show. Thanks!
